### ISSUE TITLE
Menu is unfocus

### ISSUE THREAD

**Issue by mshzsh** [hyprwm/Hyprland#1113]
The programs menu is unfocus when opened : 

https://user-images.githubusercontent.com/41986782/204129088-f850af62-04de-488e-ad61-96d323a5618d.mp4

**Comment by vaxerski** [comment]
input settings and what app is that to reproduce? the not blender one, vs code?

**Comment by mshzsh** [comment]
input settings:
```
input {
    kb_layout=us,per
    kb_options=grp:alt_shift_toggle
    numlock_by_default=1
    follow_mouse=0
    float_switch_override_focus=0

    touchpad {
        natural_scroll=no
    }
}
```
Yes vscode

**Comment by vaxerski** [comment]
Weird. Can't repro with your exact config. Worth noting I used dolphin -> right click on a file -> properties and not the vscode picker, as that one tiles for me.

**Comment by mshzsh** [comment]
On my computer, this problem exists in almost all programs except Thunar.
Do I need to upload my config file?

**Comment by vaxerski** [comment]
The rest of your config shouldn't matter.

**Comment by mshzsh** [comment]
Couldn't find a solution to fix this issue?

**Comment by vaxerski** [comment]
I am guessing no, and then I went on to fix other stuff and forgot.

**Comment by mshzsh** [comment]
I hope this issue will be solved in the next update. This is the only problem I have right now :)

**Comment by truongan** [comment]
I also have the exact problem with vscode.

**Comment by vaxerski** [comment]
cool, but it still is tiled and completely OK for me, and I can't repro.

**Comment by mshzsh** [comment]
I have sent my config file, if you have time, test it and see if the menu of programs (except for file managers programs) gets opacity and dim or not.
[hyprland.txt](https://github.com/hyprwm/Hyprland/files/11284976/hyprland.txt)

**Comment by mshzsh** [comment]
Set `dim_inactive=true` and `inactive_opacity=0.7` then right click on the VS Code or blender or ... screen or open the top menu to notice this bug :nerd_face:

**Comment by vaxerski** [comment]
oh, got it, you're running them in xwayland. That's why, I run them in native wayland (which you should...)

**Comment by vaxerski** [comment]
fixed in cde7f79af0d084cba585bb56f9302afab5a216f9
